packages/account/src/Sections/Profile/PhoneVerification/confirm-phone-number.tsx

```diff
@@ -54,14 +53,20 @@ const ConfirmPhoneNumber = observer(({ show_confirm_phone_number, setOtpVerifica
 
     useEffect(() => {
         if (email_otp_error) {
+            trackPhoneVerificationEvents({
+                action: 'error',
```

Comment:
@suisin-deriv  screenshots attached in the PR shows action:set_phone _number_error? can you calrify which is correct one?
same for other error scenarios.

Reply:
let me update the screen shots 🙏 The requirement was given to put action as error for any error coming through 😁